test/test_subsystem.py

```diff
@@ -126,6 +126,23 @@ def test_wedge_partitions():
     ])
 
 
+
```

Comment:
It might be good to have some separate test cases that check the output of ``purview_disconnection_partitions`` directly.